edit/kubevirt.io.virtualmachine/VirtualMachineSSHKey.vue

```diff
@@ -215,8 +217,8 @@ export default {
     />
 
     <ModalWithCard
-      ref="newSSH"
-      name="newSSH"
+      :ref="randomStr"
+      :name="randomStr"
```

Comment:
Why do you need a random string, when adding ssh credentials you should only be able to click one at a time ？

Reply:
this component should have random refName since it is in ArrayGroup(ArrayList),